config/auth.ts

```diff
@@ -0,0 +1,109 @@
+/**
+ * Config source: https://git.io/JY0mp
+ *
+ * Feel free to let us know via PR, if you find something broken in this config
+ * file.
+ */
+
+import { AuthConfig } from '@ioc:Adonis/Addons/Auth'
+
+/*
+|--------------------------------------------------------------------------
+| Authentication Mapping
+|--------------------------------------------------------------------------
+|
+| List of available authentication mapping. You must first define them
+| inside the `contracts/auth.ts` file before mentioning them here.
+|
+*/
+const authConfig: AuthConfig = {
+  guard: 'api',
+  guards: {
+    /*
+    |--------------------------------------------------------------------------
+    | OAT Guard
+    |--------------------------------------------------------------------------
+    |
+    | OAT (Opaque access tokens) guard uses database backed tokens to authenticate
+    | HTTP request. This guard DOES NOT rely on sessions or cookies and uses
+    | Authorization header value for authentication.
+    |
+    | Use this guard to authenticate mobile apps or web clients that cannot rely
+    | on cookies/sessions.
+    |
+    */
+    api: {
+      driver: 'oat',
```

Comment:
Any special thoughts on this driver?

Reply:
This driver uses database-backed tokens to authenticate HTTP requests and also I can use this driver to authenticate mobile apps or web clients that do not rely on cookies/sessions.

Reply:
So any difference on this versus JWTs? Or you got any thoughts on that?

Reply:
An OAT has a format that is not intended to be read by anyone hence the word opaque in OAT, it is only the issuer that knows the format of this token while a JWT has readable content as anyone can decode the token and read the information in it.